airflow/models/connection.py

```diff
@@ -43,6 +47,30 @@ def parse_netloc_to_hostname(*args, **kwargs):
     return _parse_netloc_to_hostname(*args, **kwargs)
 
 
+def sanitize_conn_id(conn_id: str | None) -> str | None:
+    """
+    Sanitizes the connection id and allows only specific characters to be within. Namely,
+    it allows alphanumeric characters plus the symbols @,#,$,%,&,!,-,_,. and () from 1
+    and up to 200 consecutive matches.
+
+    The character selection is such that it prevents the injection of javascript or
+    executable bits in order to avoid any awkward behavior in the front-end.
+
+    :param conn_id: The connection id to sanitize.
+    :return: the sanitized string, `None` otherwise.
```

Comment:
What does “otherwise” means? Should this be _the input string if it does not contain forvidden characters, otherwise None_?

Reply:
I will rephrase it in the final commit.